printer_mods/wile-e1/Deck_Panel_Support_Clips/readme.md

```diff
@@ -0,0 +1,13 @@
+### Deck Panel Support Clips
+
+These are simple panel support clips to help better support the lower deck panels of a V2.4 and other similar models.  They sandwich the deck sheet between an upper and lower support and are held in with M3 SHCS or M3 BHCS.
+
+<img src="G:\Dropbox\github\VoronUsers\printer_mods\wile-e1\Deck_Panel_Support_Clips\deck_clip_top.png" alt="deck_clip_top" style="zoom:80%;" />
```

Comment:
These image references are broken, please update them to reference the images you uploaded with your mod, e.g.

    ![deck_clip_top](deck_clip_top.png)

Reply:
Image links in readme.md are now repaired.